Implement RFD 24 for alternative DynamoDB event indexing (#6583)
gravitational/teleport: CHANGELOG.md

# Changelog

## 6.2

This release of teleport contains minor features and bugfixes.

* Changed DynamoDB events backend indexing strategy. [#6583](https://github.com/gravitational/teleport/pull/6583)

>   Warning! This will trigger a data migration on the first start after upgrade. For optimal performance perform this migration with only one auth server and no other nodes online. It may take some time and progress will be periodically written to syslog. Once Teleport starts and is accessible via Web UI, the rest of the cluster may be started.

Reviewer: ```suggestion
  Warning! This will trigger a data migration on the first start after upgrade. For optimal performance perform this migration with only one auth server online. It may take some time and progress will be periodically written to the auth server log. Once Teleport starts and is accessible via Web UI, the rest of the cluster may be started.
```

Author: Updated
